### ISSUE TITLE
请问大佬项目还在维护吗

### ISSUE THREAD

**Issue by Ginker** [jarryxy/wozaixiaoyuanAuto#1]
项目还在维护吗大佬，我在校园小程序好像更换鉴权机制了

**Comment by jarryxy** [comment]
是的 它更新了，项目我也更新了 新版的token换成了session 这个我用一个月了没有过期 而且官方增加了密码机制 我们可以利用这个获取到session 我写了一个api可以获取这个  http://yuencode.cn:5003/login?username=这里写电话号码&amp;password=这里写密码 注意我在校园设置密码要设置两次 不然会提示密码错误 具体原因我不清楚 错了五次会被封一天 但是微信还是可以正常登陆的 新版代码我没上传  你可以利用我的网站进行打卡 wozaixiaoyuannew.yuencode.cn 如果你有空可以加一个 如果session过期 使用密码登陆的获取新的session并上传至数据库 直至下次过期在重新获取 ------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年9月2日(星期四) 晚上10:22
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[desire599/wozaixiaoyuanAuto] 请问大佬项目还在维护吗 (#1)

**Comment by Ginker** [comment]
真的太感谢了，这是一个非常了不起的项目，比其他要手动配置的更加自动化

**Comment by Ginker** [comment]
尊敬的大佬，您的api “http://yuencode.cn:5003/login?username=这里写电话号码&amp;password=这里写密码”  好像不能用了，打开一直失败

**Comment by jarryxy** [comment]
截图发我看看哈，那个软件也可以获取到jwsession的，不过需要使用电脑 ------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年9月7日(星期二) 晚上8:23
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [desire599/wozaixiaoyuanAuto] 请问大佬项目还在维护吗 (#1)

**Comment by Ginker** [comment]
截图 http://129.226.180.250/1.png

**Comment by jarryxy** [comment]
你把你的网址发我看看，我看你前一条消息怎么有个amp;------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年9月7日(星期二) 晚上9:13
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [desire599/wozaixiaoyuanAuto] 请问大佬项目还在维护吗 (#1)

**Comment by Ginker** [comment]
http://yuencode.cn:5003/login?username=19861912913&amp;password=yong013579...
我加上了自己的账号和密码

**Comment by jarryxy** [comment]
http://yuencode.cn:5003/login?username=19861912913&amp;password=yong013579 这样的。amp;是没有的，然后我刚才试了你这个密码错误了，你在我在校园的小程序改个两三次，不然就提示错误，具体原因我也不清楚------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年9月7日(星期二) 晚上9:30
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [desire599/wozaixiaoyuanAuto] 请问大佬项目还在维护吗 (#1)

**Comment by jarryxy** [comment]
实在不行，你用那个获取软件，打开这个软件，电脑微信登陆小程序点开我在校园登陆就可以获取到，这个是通过抓包实现的，是绝对没有问题的，那个网址是通过发送请求解析的，官方没有提供这种方式，所以会出现密码错误的问题，你报错是因为参数写错了哈

---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年9月7日(周二) 晚上9:13
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [desire599/wozaixiaoyuanAuto] 请问大佬项目还在维护吗 (#1)




 
截图 http://129.226.180.250/1.png

**Comment by jarryxy** [comment]
我这边点开是正确的，返回的结果是密码错误5次，24小时后再尝试。编码解码一下就能看出来，你看看图片，并没有你发给我那种报错呀。密码错误5次，微信还是可以登陆的


你看看我发的截图，如果你会弄的话，可以把微信控制台打开直接可以看到，方法有很多种，都可以获取到的






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年9月7日(周二) 晚上8:23
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [desire599/wozaixiaoyuanAuto] 请问大佬项目还在维护吗 (#1)




 
尊敬的大佬，您的api “http://yuencode.cn:5003/login?username=这里写电话号码&amp;amp;password=这里写密码”  好像不能用了，打开一直失败

**Comment by Ginker** [comment]
我这里网络不稳定，梯子也很差，而且不好发送图片，请问可以发送一下邮箱吗  1144983626@qq.com

**Comment by Ginker** [comment]
大佬你好，我想利用您这个旧的重新弄一下健康打卡和日检日报，请问可不可以看一下旧的数据库表单，我看到好像要有数据库的读取。万分感谢！！

**Comment by jarryxy** [comment]
你的邮箱是多少，新版的我直接发给你好了，如果是个人使用的话，你直接把地点跟session给我，我给你添加到数据库中，





---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年9月22日(周三) 晚上8:02
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [desire599/wozaixiaoyuanAuto] 请问大佬项目还在维护吗 (#1)




 
大佬你好，我想利用您这个旧的重新弄一下健康打卡和日检日报，请问可不可以看一下旧的数据库表单，我看到好像要有数据库的读取。万分感谢！！

**Comment by Ginker** [comment]
邮箱 1144983626@qq.com
